src/content/components/TreeView.js

```diff
@@ -67,7 +69,7 @@ class TreeViewRowFixedColumns extends Component {
     const { node, columns, index, selected, highlightString } = this.props;
     const evenOddClassName = (index % 2) === 0 ? 'even' : 'odd';
     return (
-      <div className={`treeViewRow treeViewRowFixedColumns ${evenOddClassName} ${selected ? 'selected' : ''}`} style={{height: '16px'}} onClick={this._onClick}>
+      <div className={`treeViewRow treeViewRowFixedColumns ${evenOddClassName} ${selected ? 'selected' : ''}`} style={{height: '16px'}} onMouseDownCapture={this._onClick}>
```

Comment:
I changed these to mouse down capture so that the context menu would select the row. It felt weird not to be able to select the row when I was using the context menu.